interfaces/cython/cantera/examples/onedim/adiabatic_flame.py

```diff
@@ -33,11 +29,12 @@
 f.solve(loglevel=loglevel, auto=True)
 
 # Solve with the energy equation enabled
-if h5:
-    f.write_hdf('h2_adiabatic.h5', group='mix', mode='w',
+try:
+    # save to HDF container file if h5py is installed
+    f.write_hdf('adiabatic_flame.h5', group='mix', mode='w',
                 description='solution with mixture-averaged transport')
-else:
-    f.save('h2_adiabatic.xml', 'mix',
+except:
```

Comment:
Here and in similar cases. The use of the bare `except` clause is generally discouraged.
```suggestion
except ImportError:
```